GLTFSDK.Test/Source/GLBResourceWriterTests.cpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "stdafx.h"
+
+#include <GLTFSDK/GLBResourceWriter.h>
+
+#include "TestUtils.h"
+
+using namespace glTF::UnitTest;
+
+namespace Microsoft
+{
+    namespace  glTF
+    {
+        namespace Test
+        {
+            GLTFSDK_TEST_CLASS(GLBResourceWriterTests)
+            {
+                GLTFSDK_TEST_METHOD(GLBResourceWriterTests, WriteBufferView_Empty)
+                {
+                    auto streamWriter = std::make_shared<const StreamReaderWriter>();
+                    GLBResourceWriter writer(streamWriter);
+                    std::string uri = "glb";
+                    writer.Flush("", uri);
+
+                    std::vector<char> output(100);
```

Comment:
This is for writing another external file such as a png, why are you doing this here?

Reply:
I just needed a way to access the output stream in the cache. Now i'm using the streamWriter directly